guide/matrix-vs-discord.md

```diff
@@ -112,10 +114,14 @@ Note that Matrix does not (and cannot, due to its decentralized nature) paywall
 
 [^2]: For Discord "servers" and for Matrix rooms.
 
-[^3]: Note that Matrix has no concept of "friends" or "contacts," although the DM list can serve the same purpose.
+[^3]: Note that Matrix has no concept of "friends" or "contacts", although the DM list can serve the same purpose. However, user-based contact ignoring exists on Matrix, using two different methods. The former method prevents any and all messages from reaching the recipient, and the latter method hides them client-side by interpreting an ignore list.
```

Comment:
Quote and comma: I follow the [MLA style](https://style.mla.org/punctuation-and-quotation-marks/). Also you should specify the methods (especially given that the latter is a lab feature).

Reply:
The messages would reach the recipient even if we are talking encrypted rooms they would just not get the keys.